client/client_app/views/index.py

```diff
@@ -81,11 +140,29 @@ def register():
                         form.password.data)
         db.session.add(new_user)
         db.session.commit()
-        flash('Thanks for registering', 'succes')
+        flash('Thanks for registering', 'success')
         return redirect(url_for('index'))
     return render_template('register.html', form=form)
 
 
+@app.route('/edit', methods=['GET', 'POST'])
+@is_logged
+def edit():
+    user_id = session['logged_in']
+    current_user = User.query.get(user_id)
+    form = edit_form.EditForm(request.form)
```

Comment:
As this is a edit form, you need to populate form by current values stored in DB. Please check `obj` argument in [Form](https://wtforms.readthedocs.io/en/latest/forms.html#wtforms.form.Form) class.

Reply:
I won't solve it to the end of this sprint. Task will be reopened and fixed in next one.